### PR TITLE
More updates

### PR DESCRIPTION
**Pull request by helveticade**
Language updates + a few visual tweaks

### CODE REVIEW

File: src/views/EditView.js [hunk 1]
```diff
@@ -17,8 +17,8 @@
       this.on("viewDeactivate",this.viewDeactivate);
       this.model.bind("all", this.addAll, this);
       this.confirmBackNav = {
-        title: "Confirm navigation",
-        subtitle: "Discard any changes?",
+        title: "You have an unsaved item",
+        subtitle: "Go back and discard this new item?",
```
Comment (devgeeks) [inline src/views/EditView.js]: This is for editing as well as adding... "new item" might not be what you are after...

File: src/views/SettingsView.js [hunk 2]
```diff
@@ -82,8 +82,8 @@
       this.on("viewActivate",this.viewActivate);
       this.on("viewDeactivate",this.viewDeactivate);
       this.confirmBackNav = {
-        title: "Are you sure?",
-        subtitle: "Discard any changes?",
+        title: "You have unsaved changes",
+        subtitle: "Go back and discard your changes?",
```
Comment (devgeeks) [inline src/views/SettingsView.js]: I said "any" changes, because they might or might not have changed anything.

File: www/img/Note.svg [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (devgeeks) [inline www/img/Note.svg]: Note.svg - shouldn't this still be General.svg?

Comment (helveticade) [inline www/img/Note.svg]: Yeah, this is unused. Apologies.

**Pull request merged**